### ISSUE TITLE
add support for noise cancelation

### ISSUE THREAD

**Issue by davehorton** [jambonz/jambonz-feature-server#810]
add support for noise cancellation, possibly using a library like [rnnoise](https://gitlab.xiph.org/xiph/rnnoise)

**Comment by vdharashive** [comment]
@davehorton this would interesting feature , but would it have any latency introduce? and how expensive will be on feature server ?

**Comment by davehorton** [comment]
yes, this is an open source library we could use and would also like to consider commercial options like [krisp](https://krisp.ai/developers/#technologies)
